### ISSUE TITLE
Mxnet : test and validation accuracy during training ?

### ISSUE THREAD

**Issue by Shiro-LK** [apache/incubator-mxnet#6666]
hello,

I am trying to have the training, the test and validation accuracy at the end of each epoch. To do that, I indicate in the fit function the validation accuracy with the eval_data parameters. But I have a problem with the test accuracy. I wanted to add this option in the eval_end_callbacks thank to the function mx.callback.Speedometer. I do not have an error but I do not see and do not know how to get the result.

> eval_end_callbacks = [mx.contrib.tensorboard.LogMetricsCallback(evaluation_log), mx.callback.Speedometer(mod.score(test_dataiter,metric))]

How can I solve this problem ?

**Comment by ashwin-cognitiv** [comment]
I've been having the same issue. Besides repeatedly swapping the eval_data with the test_iter data to compare it with the training data, I'm still not sure how to print all 3 at the end of the epoch.

**Comment by chao1224** [comment]
@Shiro-mx  Please take a look at this #6695

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
